napari/layers/_shapes_layer/model.py

```diff
@@ -545,18 +545,19 @@ def _refresh(self):
         self._need_display_update = True
         self._update()
 
-    def _set_view_slice(self, indices=None):
-        """Set the shape mesh data to the view.
+    def _set_view_slice(self, indices):
+        """Sets the view given the indices to slice with.
```

Comment:
do not write in 3rd-person
```suggestion
        """Set the view given the indices to slice with.
```

Reply:
fixed